group.lisp

```diff
@@ -542,6 +542,19 @@ The windows will be moved to group \"^B^2*~a^n\"
             (message "Canceled"))
         (message "There's only one group left"))))
 
+(defcommand gkill-other () ()
+  "Kill other groups. All windows in other groups are migrated
+to the current group."
+  (let* ((current-group (current-group))
+         (groups (remove-if (lambda (g) (eq g current-group))
+                            (screen-groups (current-screen)))))
+    (if (> (length groups) 0)
```

Comment:
What do you think about removing the messages so that there is only one execution path?

```lisp
(let* ((current-group (current-group))
         (groups (remove current-group
                         (screen-groups (current-screen)))))
    (dolist (dead-group groups)
      (kill-group dead-group current-group)))
```

Reply:
yup, i think removing the message make it more concise and straightforward :smile: